### DISCUSSION TITLE
Weird error when building and starting OS

### DISCUSSION THREAD

**Discussion by KernelFox0** [CosmosOS/Cosmos#2136] [Q&A]
What is this error? Can't boot cosmos because it always shows this. Reinstalled, recompiled, using latest visual studio 2019. Can't continue my project because of this. Using latest dev kit.
![error1](https://user-images.githubusercontent.com/65031582/155839786-c4a45b98-28c9-466e-a61a-bc4dee57b067.png)
![error2](https://user-images.githubusercontent.com/65031582/155839788-437a0eb4-ba17-4e24-b556-21c04ede7dbe.png)

**Comment by quajak** [comment]
There is a weird non-deterministic bug with the current devkit, if you encounter this error try reinstalling until it works.
